packages/lesswrong/components/common/CommentsListCondensed.tsx

```diff
@@ -55,7 +55,7 @@ const CommentsListCondensed = ({label, terms, initialLimit, itemsPerPage, showTo
 const CommentsListCondensedComponent = registerComponent(
   'CommentsListCondensed',
   CommentsListCondensed,
-  {styles}
+  {styles, stylePriority: 1},
```

Comment:
This doesn't look like it's used on the All Posts page, why this change?

Reply:
A different PR changed the stylePriority of SectionTitle, which broke this.